dev-requirements.txt

```diff
@@ -5,3 +5,4 @@ requests==2.5.1
 responses==0.3.0
 wheel
 twine
+Django==1.8.5
```

Comment:
I think this backend will work even earlier versions of django. So I don't know can I test (automated) it against multiple versions of django. In that case we can add that whole range of versions here (like `Django>=1.4.0,Django<1.9`).